soccer/gameplay/tactics/test_cost_function.py

```diff
@@ -0,0 +1,15 @@
+import robocup
+import constants
+import math
+from typing import Tuple, Callable
+
+
+## A test cost function for passes that should have a minima at (0.915239, 1.38493)
+# Takes in an array of 2 cooridnates
+def create_test_cost_function(
```

Comment:
You can also use `partial` to bind args to a function.

I do it [here](https://github.com/RoboJackets/robocup-software/blob/staging/soccer/gameplay/evaluation/passing_positioning.py#L116). Documentation is [here](https://docs.python.org/3/library/functools.html#functools.partial)